repos.bzl

```diff
@@ -29,6 +34,57 @@ def configure(minimum_bazel_version = None, rbe_name = "rbe_default", go_version
     protobuf_deps()  # No options
 
     go_rules_dependencies()  # No options
+
+    go_download_sdk(
```

Comment:
Please make this optional as I believe it forces everyone to use 1.15beta1.

For example:

```python
EXTRA_VERSIONS = {"go1.15beta1": {
  "darwin_amd64": ("go1.15beta1.darwin-amd64.tar.gz", "4ee49...bd7e"),
  "freebsd_386": ("go1.15beta1.freebsd-386.tar.gz", "77bc...5c94"),
  ...
}}

def configure(... extra_version=None):
  ...
  if extra_version:   
      go_download_sdk(name="go_sdk", sdks=EXTRA_VERSIONS[extra_version])
      go_register_toolchains(go_version = extra_version, nogo = nogo)
  else 
      go_register_toolchains(go_version = go_version, nogo = nogo)
```

Reply:
I like this! Will do.

Reply:
Fixed up!